packages/eas-cli/schema/metadata-0.json

```diff
@@ -256,112 +266,143 @@
+            "$ref": "#/definitions/apple/AppleRating",
+            "description": "Does the app contain profanity or crude humor?"
           },
           "sexualContentGraphicAndNudity": {
-            "$ref": "#/definitions/apple/AppleRating"
+            "$ref": "#/definitions/apple/AppleRating",
+            "description": "Does the app contain graphic sexual content and nudity?"
           },
           "sexualContentOrNudity": {
-            "$ref": "#/definitions/apple/AppleRating"
-          },
-          "seventeenPlus": {
-            "type": "boolean"
+            "$ref": "#/definitions/apple/AppleRating",
+            "description": "Does the app contain sexual content or nudity?"
           },
           "horrorOrFearThemes": {
-            "$ref": "#/definitions/apple/AppleRating"
+            "$ref": "#/definitions/apple/AppleRating",
+            "description": "Does the app contain horror or fear themes?"
           },
           "matureOrSuggestiveThemes": {
-            "$ref": "#/definitions/apple/AppleRating"
-          },
-          "unrestrictedWebAccess": {
-            "type": "boolean"
+            "$ref": "#/definitions/apple/AppleRating",
+            "description": "Does the app contain mature or suggestive themes?"
           },
           "violenceCartoonOrFantasy": {
-            "$ref": "#/definitions/apple/AppleRating"
+            "$ref": "#/definitions/apple/AppleRating",
+            "description": "Does the app contain cartoon or fantasy violence?"
           },
           "violenceRealisticProlongedGraphicOrSadistic": {
-            "$ref": "#/definitions/apple/AppleRating"
+            "$ref": "#/definitions/apple/AppleRating",
+            "description": "Does the app contain prolonged graphic or sadistic realistic violence?"
           },
           "violenceRealistic": {
-            "$ref": "#/definitions/apple/AppleRating"
+            "$ref": "#/definitions/apple/AppleRating",
+            "description": "Does the app contain realistic violence?"
+          },
+          "unrestrictedWebAccess": {
+            "type": "boolean",
+            "description": "Does your app contain unrestricted web access, such as with an embedded browser?"
+          },
+          "gambling": {
+            "type": "boolean",
+            "description": "Does your app contain gambling?"
+          },
+          "kidsAgeBand": {
+            "oneOf": [
+              {
+                "type": "null"
+              },
+              {
+                "$ref": "#/definitions/apple/AppleKidsAge"
+              }
+            ],
+            "description": "The Kids category is a great way for people to easily find apps that are designed for children. Keep in mind that once your Made for Kids app is approved by App Review, the app and all subsequent updates will need to follow the Kids category guidelines unless you create a new app in App Store Connect.\n@see https://developer.apple.com/news/?id=091202019a",
```

Comment:
Still reads a bit markety to me, but doesn't matter much.

Reply:
Just fixed that :)